traitator/tests/test_qctraits.py

```diff
@@ -20,14 +20,14 @@ def _find_nan_rows(table: pd.DataFrame, cols: [str]) -> pd.DataFrame:
 class QCTests(TestCase):
     def setUp(self):
         self.fp_guitar = 'traitator/tests/data/trait_data_n10906.csv'
-        self.fp_tresor = 'data/traitdata_tresor.csv'
```

Comment:
please don't use a different file! Only if you replace the existing file with your version the test system makes sense.

Reply:
that means, you want me to overwrite the file?

Reply:
why does the file name "traitdata_tresor" has a ."csv" appended to it, although the file is actually a tsv file?